src/test/java/org/apache/maven/shared/model/fileset/util/FileSetUtilsTest.java

```diff
@@ -269,23 +262,19 @@ void testDeleteDontFollowSymlinksButDeleteThem() throws Exception {
         assertFalse(new File(directory, "dir1").exists(), "included directory has not been deleted");
     }
 
-    private boolean createSymlink(File target, File link) throws InterruptedException, CommandLineException {
-
+    private void createSymlink(File target, File link) {
         if (link.exists()) {
             link.delete();
         }
 
-        Commandline cli = new Commandline();
-        cli.setExecutable("ln");
-        cli.createArg().setValue("-s");
-        cli.createArg().setValue(target.getPath());
-        cli.createArg().setValue(link.getPath());
-
-        int result = cli.execute().waitFor();
-
-        linkFiles.add(link);
-
-        return result == 0;
+        try {
+            Files.createSymbolicLink(link.toPath(), target.toPath());
+            linkFiles.add(link);
```

Comment:
Do we need to track create links in @TempDir?, should be removed by JUnit

Reply:
good point. done